### PR TITLE
fix(v2): fix toggle named projection

### PR DESCRIPTION
**Pull request by Varixo**
This PR fixes toggling named projection with an interaction

### CODE REVIEW

File: packages/qwik/src/core/v2/client/dom-container.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ import {
   vnode_setProp,
   type VNodeJournal,
 } from './vnode';
-import { vnode_diff } from './vnode-diff';
+import { isSlotProp, vnode_diff } from './vnode-diff';
```
Comment (igalklebanov) [inline packages/qwik/src/core/v2/client/dom-container.ts:67]: are we sure we want `isSlotProp` to sit @ `vnode-diff`? 🤔

File: packages/qwik/src/core/v2/client/dom-container.ts [hunk 2]
```diff
@@ -289,7 +289,7 @@ export class DomContainer extends _SharedContainer implements IClientContainer,
       vNode[VNodeProps.flags] |= VNodeFlags.Resolved;
       for (let i = vnode_getPropStartIndex(vNode); i < vNode.length; i = i + 2) {
         const prop = vNode[i] as string;
-        if (!prop.startsWith('q:')) {
+        if (isSlotProp(prop)) {
```
Comment (igalklebanov) [inline packages/qwik/src/core/v2/client/dom-container.ts:292]: 👨‍🍳 💋

File: packages/qwik/src/core/v2/client/dom-container.ts (hunk 1 above)
Comment (Varixo) [reply packages/qwik/src/core/v2/client/dom-container.ts:67]: yeah, I will think about it later

Comment (Varixo) [reply packages/qwik/src/core/v2/client/dom-container.ts]: I moved this into separate file

**Pull request merged**